components/gfx/text/glyph.rs

```diff
@@ -71,7 +71,7 @@ pub type GlyphId = u32;
 
 // TODO: make this more type-safe.
 
-const FLAG_CHAR_IS_SPACE: u32 = 0x40000000;
+const FLAG_CHAR_IS_JUSTIFICATION_OPPORTUNITY: u32 = 0x40000000;
```

Comment:
Maybe "[word separator](https://drafts.csswg.org/css-text/#word-separator)" instead of "justification opportunity"? Since `text-justify: inter-character` may create more justification opportunities

Reply:
Okay. I've done this as well as add the rest of the word separators.